src/main/java/Tinhe.java

```diff
@@ -0,0 +1,14 @@
+public class Tinhe {
+    public String tinhe(){
+        double e=2;
+        long giaithua = 1;
+        long i=2;
+        double E = 2.718281828;
+        while(Math.abs(E-e)>0.000001){
```

Comment:
Viết thiếu khoảng trống ở các biểu thức
Ví dụ

```suggestion
        while(Math.abs(E-e) > 0.000001){
```